corehq/apps/userreports/ui/widgets.py

```diff
@@ -5,4 +5,8 @@
 class JsonWidget(forms.Textarea):
 
     def render(self, name, value, attrs=None):
+        if isinstance(value, basestring):
+            # It's probably invalid JSON
```

Comment:
I was just thinking about this some more.  I don't think the `not is_json` will do much.  We don't currently distinguish between a simple python string (`"foo"`) and a python string of a json string (`'"foo"'`), as `try: return json.loads(var) except: return var` will return `"foo"` in either case.

If we're concerned about this field's handling of json strings, we should probably revisit the implementation of JsonField, but if not, I think the assumptions I made here are okay (particularly as this only affects how the value is rendered in a form).